MAUI/Badge-View/Predefined-symbols.md

```diff
@@ -0,0 +1,66 @@
+---
+
+layout: post
+title: Predefined symbols of Syncfusion Badge view control for .NET MAUI
+description: Learn how to set Syncfusion badge view predefined symbols and types of badge icons in .NET MAUI platform
+platform: maui
+control: SfBadgeView
+documentation: ug
+
+---
+
+# Predefined symbols in .NET MAUI Badge View (SfBadgeView)
+
+The badge icon has changed using the `Icon` property. Badge icons will be visible when you do not set the badge text. The badge supports the following types of `Icon`
```

Comment:
The badge icon has been changed using the `Icon` property. Badge icons will be visible when you do not set the badge text. The badge supports the following types of `Icon`.

Keep full stop at the end.